DigitalLearningSolutions.Web/ViewModels/Common/SearchablePage/BaseSearchablePageViewModel.cs

```diff
@@ -33,17 +38,16 @@ public abstract class BaseSearchablePageViewModel
             SearchString = searchString;
             FilterString = filterString;
             Page = page;
+            SortEnabled = sortEnabled;
+            FilterEnabled = filterEnabled;
             Filters = new List<FilterViewModel>();

```

Comment:
Did we want this renamed to FilterBy to match the SortBy, or are we keeping it as Filters to highlight the fact it is a list? Same for the BaseJavascriptFilterableViewModel

Reply:
Not sure what you mean here, the corresponding sort property to Filters is SortOptions. FilterString should perhaps be changed to FilterBy though.